src/debugger.ts

```diff
@@ -60,7 +60,7 @@ function changeConfigForSip(config: vscode.DebugConfiguration, executableFieldNa
 		// Run the command in a SIP-patched shell, that way everything that runs in the original command will be SIP-patched
 		// on runtime.
 		config[executableFieldName] = `echo '${escapedCommand}' | ${DYLD_ENV_VAR_NAME}=${libraryPath} ${sh} -is`;
-	} else if (executionInfo.patchedPath !== null) {
+	} else if (executionInfo.patchedPath != null) {
```

Comment:
```suggestion
	} else if (executionInfo.patchedPath) {
```

No need to check with equals, `if` takes care of it.